src/stylesheets/base/_colors.scss

```diff
@@ -0,0 +1,14 @@
+$colors: (
+  cool-charcoal:        #252224,
+  warm-charcoal:        #504949,
+  deep-blue:            #184D66,
+  emerald:              #0E5254,
+  teal:                 #1793AD,
+  mint:                 #D3E7E2,
+  grey:                 #E2EBE9,
+  white:                #FFFFFF,
+  yellow:               #F2C84D,
+  orange:               #C06D3A,
+  coral:                #EA4446,
+  gradient-background:  linear-gradient(#184D66 3.08%, #D3E7E2 94.99%)
+);
```

Comment:
GitHub is giving you this 🚫 ↩️  thing because your files don't end in a blank line. By convention we add one (google "final newline" to learn all about it). To automate this, most (all?) of us use the tool [EditorConfig](https://editorconfig.org/). A `.editorconfig` is placed at the root of the site, and a text editor plugin automatically makes small formatting adjustments on save. Some examples: https://github.com/search?q=org%3Avigetlabs+editorconfig&type=Code. Typical is something like

```
root = true

[*]
end_of_line = lf
trim_trailing_whitespace = true
insert_final_newline = true
indent_style = space
indent_size = 2

[*.md]
trim_trailing_whitespace = false

[*.php]
indent_style = tab
indent_size = 4
```

Reply:
Cool, and good to know. VS Code has a setting for this and it was switched off for some reason, but I should be good to go now.